### PR TITLE
Add gd-sprest + gd-sprest-bs w/ npm auto-update

### PR DESCRIPTION
**Pull request by gunjandatta**
Request to add a new package to the CDN. Reference [this pr](https://github.com/cdnjs/cdnjs/pull/13949)

### CODE REVIEW

File: packages/g/gd-sprest-bs.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "bugs": {
+    "url": "https://github.com/gunjandatta/sprest-bs/issues"
+  },
```
Comment (MattIPv4) [inline packages/g/gd-sprest-bs.json:27]:
```suggestion
```
We don't need the `bugs` property.

File: packages/g/gd-sprest-bs.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "rest",
+    "bootstrap"
+  ],
+  "homepage": "https://dattabase.com/",
+  "npmName": "gd-sprest-bs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (MattIPv4) [inline packages/g/gd-sprest-bs.json:21]:
```suggestion
        "*.@(js|ts)"
```
[Use a more specific glob.](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.%40%28js%7Cts%29&matches=true&tests=%2F%2F%20Imported%20from%20gd-sprest-bs%20NPM%20package&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.min.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.structure.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.structure.min.css&tests=lib%2Fjquery-ui-1.12.1.custom%2Findex.html&tests=dist%2Fgd-sprest-bs-icons.js&tests=dist%2Fgd-sprest-bs-icons.min.js&tests=dist%2Fgd-sprest-bs.js&tests=dist%2Fgd-sprest-bs.min.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fjquery-ui.min.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fexternal%2Fjquery%2Fjquery.js&tests=test.js&tests=lib%2Fjquery-ui-1.12.1.custom%2Fpackage.json&tests=package.json&tests=README.md&tests=src%2Fstyles.scss&tests=src%2Fwebparts%2Ftabs%2Fstyles.scss&tests=.%2F%40types%2Fcomponents%2Fdatetime.d.ts&tests=src%2Fcomponents%2Fdatetime.ts&tests=.%2F%40types%2Fcomponents%2Ffield.d.ts&tests=src%2Fcomponents%2Ffield.ts&tests=dist%2Fgd-sprest-bs.d.ts&tests=.%2F%40types%2Fwebparts%2Fhelper.d.ts&tests=src%2Fwebparts%2Fbase%2Fhelper.ts&tests=src%2Findex-icons.ts&tests=.%2F%40types%2Fcomponents%2Findex.d.ts&tests=.%2F%40types%2Findex.d.ts&tests=.%2F%40types%2Fwebparts%2Findex.d.ts&tests=src%2Fcomponents%2Findex.ts&tests=src%2Findex.ts&tests=src%2Fwebparts%2Fbase%2Findex.ts&tests=src%2Fwebparts%2Findex.ts&tests=src%2Fwebparts%2Flist%2Findex.ts&tests=src%2Fwebparts%2FlistFields%2Findex.ts&tests=src%2Fwebparts%2Fsearch%2Findex.ts&tests=src%2Fwebparts%2Ftabs%2Findex.ts&tests=src%2Fwebparts%2Ftaxonomy%2Findex.ts&tests=.%2F%40types%2Fcomponents%2FlistForm.d.ts&tests=src%2Fcomponents%2FlistForm.ts&tests=.%2F%40types%2Fcomponents%2FlistFormDialog.d.ts&tests=src%2Fcomponents%2FlistFormDialog.ts&tests=.%2F%40types%2Fcomponents%2FpeoplePicker.d.ts&tests=src%2Fcomponents%2FpeoplePicker.ts&tests=.%2F%40types%2Frest.d.ts&tests=src%2Frest.ts&tests=.%2F%40types%2Fwebparts%2Fwp.d.ts&tests=src%2Fwebparts%2Fbase%2Fwp.ts&tests=src%2Fwebparts%2Flist%2Fwp.ts&tests=src%2Fwebparts%2FlistFields%2Fwp.ts&tests=src%2Fwebparts%2Fsearch%2Fwp.ts&tests=src%2Fwebparts%2Ftabs%2Fwp.ts&tests=src%2Fwebparts%2Ftaxonomy%2Fwp.ts&tests=src%2Fwebparts%2Fbase%2FwpCfg.ts&tests=src%2Fwebparts%2Flist%2FwpCfg.ts&tests=src%2Fwebparts%2FlistFields%2FwpCfg.ts&tests=src%2Fwebparts%2Fsearch%2FwpCfg.ts&tests=src%2Fwebparts%2Ftaxonomy%2FwpCfg.ts&tests=.%2F%40types%2Fwebparts%2FwpList.d.ts&tests=.%2F%40types%2Fwebparts%2FwpListFields.d.ts&tests=.%2F%40types%2Fwebparts%2FwpSearch.d.ts&tests=.%2F%40types%2Fwebparts%2FwpTabs.d.ts&tests=.%2F%40types%2Fwebparts%2FwpTaxonomy.d.ts&tests=lib%2Fjquery-ui-1.12.1.custom%2FAUTHORS.txt&tests=lib%2Fjquery-ui-1.12.1.custom%2FLICENSE.txt&tests=pnpm-lock.yaml)

File: packages/g/gd-sprest-bs.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gd-sprest-bs",
+  "filename": "dist/gd-sprest-bs-icons.min.js",
```
Comment (MattIPv4) [inline packages/g/gd-sprest-bs.json:3]:
```suggestion
  "filename": "gd-sprest-bs-icons.min.js",
```
As the auto-update config uses a base of `dist`, we don't need it here.

File: packages/g/gd-sprest.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "spo",
+    "rest"
+  ],
+  "homepage": "https://dattabase.com/",
+  "npmName": "gd-sprest",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (MattIPv4) [inline packages/g/gd-sprest.json:20]:
```suggestion
        "*.@(js|ts)"
```
[Use a more specific glob for just the files needed.](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.%40%28js%7Cts%29&matches=true&tests=%2F%2F%20Imported%20from%20gd-sprest%20NPM%20package&tests=build%2Fhelper%2Fmethods%2FaddContentEditorWebPart.js&tests=build%2Fhelper%2Fmethods%2FaddScriptEditorWebPart.js&tests=build%2Flib%2Fapps.js&tests=build%2Fmapper%2Fcustom%2Faudit.js&tests=build%2Futils%2Fbase.js&tests=build%2Futils%2Fbatch.js&tests=build%2Fhelper%2Fsp%2FcalloutManager.js&tests=clean.js&tests=build%2Flib%2FcontextInfo.js&tests=build%2Fhelper%2Fmethods%2FcreateContentType.js&tests=build%2Fhelper%2Fmethods%2FcreateDocSet.js&tests=build%2Fmapper%2Fdef.js&tests=build%2Fhelper%2Fexecutor.js&tests=build%2Fhelper%2FfieldSchemaXML.js&tests=dist%2Fgd-sprest.js&tests=dist%2Fgd-sprest.min.js&tests=build%2Flib%2Fgraph.js&tests=build%2Fmapper%2Fcustom%2Fgraph.js&tests=build%2Flib%2FgroupService.js&tests=build%2Flib%2FgroupSiteManager.js&tests=build%2Fhelper%2Fmethods%2FhasPermissions.js&tests=build%2Futils%2Fhelper.js&tests=build%2Flib%2FhubSites.js&tests=build%2Flib%2FhubSitesUtility.js&tests=build%2Fhelper%2Findex.js&tests=build%2Fhelper%2Fmethods%2Findex.js&tests=build%2Fhelper%2Fsp%2Findex.js&tests=build%2Findex.js&tests=build%2Flib%2Findex.js&tests=build%2Fmapper%2Fcustom%2Findex.js&tests=build%2Fmapper%2Findex.js&tests=build%2Fsptypes%2Findex.js&tests=build%2Futils%2Findex.js&tests=build%2Fhelper%2Fjslink.js&tests=build%2Flib%2Flist.js&tests=build%2Fhelper%2FlistForm.js&tests=build%2Fhelper%2FlistFormField.js&tests=build%2Futils%2FmethodInfo.js&tests=build%2Fhelper%2Fsp%2FmodalDialog.js&tests=build%2Flib%2Fnavigation.js&tests=build%2Fhelper%2Fsp%2Fnotify.js&tests=build%2Futils%2FoData.js&tests=build%2Fmapper%2Fcustom%2Fold.js&tests=build%2Fhelper%2Fmethods%2Fparse.js&tests=build%2Flib%2FpeopleManager.js&tests=build%2Flib%2FpeoplePicker.js&tests=build%2Fmapper%2Fcustom%2FpeoplePicker.js&tests=build%2Flib%2FprofileLoader.js&tests=build%2Fmapper%2Fcustom%2FpropertyValues.js&tests=build%2Fhelper%2Fmethods%2Frequest.js&tests=build%2Futils%2Frequest.js&tests=build%2Futils%2FrequestType.js&tests=build%2Frest.js&tests=build%2Fhelper%2FribbonLink.js&tests=build%2Fhelper%2FsbLink.js&tests=build%2Flib%2Fsearch.js&tests=build%2Fhelper%2Fmethods%2FsetContentTypeFields.js&tests=build%2Flib%2Fsite.js&tests=build%2Flib%2FsocialFeed.js&tests=build%2Fhelper%2Fsp%2Fsod.js&tests=build%2Fhelper%2FspCfg.js&tests=build%2Fhelper%2FspCfgTypes.js&tests=build%2Fsptypes%2Fsptypes.js&tests=build%2Fhelper%2Fsp%2Fstatus.js&tests=build%2Futils%2FtargetInfo.js&tests=build%2Fhelper%2Ftaxonomy.js&tests=build%2Flib%2FthemeManager.js&tests=build%2Flib%2FuserProfile.js&tests=build%2Flib%2Futility.js&tests=build%2Fmapper%2Fcustom%2Futility.js&tests=build%2Flib%2Fweb.js&tests=build%2Fhelper%2Fwebpart.js&tests=build%2Flib%2FwfInstanceService.js&tests=build%2Flib%2FwfSubscriptionService.js&tests=build%2Futils%2FxhrRequest.js&tests=dts-bundle.json&tests=package.json&tests=LICENSE.md&tests=README.md&tests=.%2F%40types%2Flib%2Fapps.d.ts&tests=.%2F%40types%2Futils%2Fbase.d.ts&tests=.%2F%40types%2Flib%2FcontextInfo.d.ts&tests=.%2F%40types%2Fhelper%2Fexecutor.d.ts&tests=.%2F%40types%2Fhelper%2FfieldSchemaXML.d.ts&tests=dist%2Fgd-sprest.d.ts&tests=.%2F%40types%2Fintellisense%2Fgraph.d.ts&tests=.%2F%40types%2Flib%2Fgraph.d.ts&tests=.%2F%40types%2Flib%2FgroupService.d.ts&tests=.%2F%40types%2Flib%2FgroupSiteManager.d.ts&tests=.%2F%40types%2Futils%2Fhelper.d.ts&tests=.%2F%40types%2Flib%2FhubSites.d.ts&tests=.%2F%40types%2Flib%2FhubSitesUtility.d.ts&tests=.%2F%40types%2Fhelper%2Findex.d.ts&tests=.%2F%40types%2Findex.d.ts&tests=.%2F%40types%2Fintellisense%2Findex.d.ts&tests=.%2F%40types%2Flib%2Findex.d.ts&tests=.%2F%40types%2Fsptypes%2Findex.d.ts&tests=.%2F%40types%2Futils%2Findex.d.ts&tests=.%2F%40types%2Fhelper%2Fjslink.d.ts&tests=.%2F%40types%2Fhelper%2FlinkInfo.d.ts&tests=.%2F%40types%2Flib%2Flist.d.ts&tests=.%2F%40types%2Fhelper%2FlistForm.d.ts&tests=.%2F%40types%2Fhelper%2FlistFormField.d.ts&tests=.%2F%40types%2Fhelper%2Fmethods.d.ts&tests=.%2F%40types%2Flib%2Fnavigation.d.ts&tests=.%2F%40types%2Flib%2FpeopleManager.d.ts&tests=.%2F%40types%2Fintellisense%2FpeoplePicker.d.ts&tests=.%2F%40types%2Flib%2FpeoplePicker.d.ts&tests=.%2F%40types%2Flib%2FprofileLoader.d.ts&tests=.%2F%40types%2Futils%2Frequest.d.ts&tests=.%2F%40types%2Futils%2FrequestType.d.ts&tests=.%2F%40types%2Frest.d.ts&tests=.%2F%40types%2Flib%2Fsearch.d.ts&tests=.%2F%40types%2Flib%2Fsite.d.ts&tests=.%2F%40types%2Flib%2FsocialFeed.d.ts&tests=.%2F%40types%2Fhelper%2Fsp.d.ts&tests=.%2F%40types%2Fhelper%2FspCfg.d.ts&tests=.%2F%40types%2Fhelper%2FspCfgTypes.d.ts&tests=.%2F%40types%2Fsptypes%2Fsptypes.d.ts&tests=.%2F%40types%2Futils%2FtargetInfo.d.ts&tests=.%2F%40types%2Fhelper%2Ftaxonomy.d.ts&tests=.%2F%40types%2Flib%2FthemeManager.d.ts&tests=.%2F%40types%2Flib%2FuserProfile.d.ts&tests=.%2F%40types%2Fintellisense%2Futility.d.ts&tests=.%2F%40types%2Flib%2Futility.d.ts&tests=.%2F%40types%2Flib%2Fweb.d.ts&tests=.%2F%40types%2Fhelper%2Fwebpart.d.ts&tests=.%2F%40types%2Flib%2FwfInstanceService.d.ts&tests=.%2F%40types%2Flib%2FwfSubscriptionService.d.ts&tests=.%2F%40types%2Futils%2FxhrRequest.d.ts&tests=pnpm-lock.yaml)

File: packages/g/gd-sprest.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+      ]
+    }
+  ],
+  "bugs": {
+    "url": "https://github.com/gunjandatta/sprest/issues"
+  },
```
Comment (MattIPv4) [inline packages/g/gd-sprest.json:26]:
```suggestion
```
`bugs` property isn't needed.

File: packages/g/gd-sprest.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gd-sprest",
+  "filename": "dist/gd-sprest.min.js",
```
Comment (MattIPv4) [inline packages/g/gd-sprest.json:3]:
```suggestion
  "filename": "gd-sprest.min.js",
```
Auto-update uses `dist` as the base, so we don't need it here.

File: packages/g/gd-sprest-bs.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gd-sprest-bs",
+  "filename": "gd-sprest-bs-icons.min.js",
```
Comment (MattIPv4) [inline packages/g/gd-sprest-bs.json:3]:
```suggestion
  "filename": "gd-sprest-bs.min.js",
```
I'm not familiar with this library, but should this file be suggested as the default instead?

File: packages/g/gd-sprest-bs.json (hunk 2 above)
Comment (MattIPv4) [reply packages/g/gd-sprest-bs.json:21]: This ^

File: packages/g/gd-sprest-bs.json (hunk 7 above)
Comment (gunjandatta) [reply packages/g/gd-sprest-bs.json:3]: No, this is correct. The bootstrap icons were later added to v4, but is still made optional due to the size of the library. Once bootstrap v5 is released, I'll bundle the icons and re-update this package definition.

Comment (MattIPv4) [reply packages/g/gd-sprest-bs.json:3]: 👍

**Pull request merged**